Kitodo/src/main/webapp/newpages/ProjekteBearbeiten.xhtml

```diff
@@ -79,7 +80,7 @@
                                                     infoClass="text_blue" showDetail="true" showSummary="true"
                                                     tooltip="true"/>
 
-                                        <!-- Box für die Bearbeitung der Details -->
+                                        <!-- Box f�r die Bearbeitung der Details -->
```

Comment:
ah! I was wondering, why this comment is marked as a change, but of course, it's the encoding.